partner_credit_limit/models/sale.py

```diff
@@ -12,7 +12,7 @@ class SaleOrder(models.Model):
 
     @api.one
     def check_limit(self):
-        partner = self.partner_id
+        partner = self.partner_invoice_id
```

Comment:
I would say no! 
You can sell product to a company OR a contact! 
What matters is how much Credit you you give either.
Either way the credit is given to the invoice account hence the partner_invoice_id